.github/workflows/ci-aarch64-nightly.yml

```diff
@@ -0,0 +1,142 @@
+# *******************************************************************************
+# Copyright 2024 Arm Limited and affiliates.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# *******************************************************************************
+
+name: "CI AArch64 (Nightly)"
+
+#* To avoid duplicate jobs running when both push and PR is satisfied, we use this:
+#* https://github.com/orgs/community/discussions/26940#discussioncomment-5686753
+on:
+  pull_request:
```

Comment:
No, I am just using it to test. Once it works for the PR, then I know that the cron job will work as well. The cron job will otherwise not trigger unless I submit this to main.

I will remove `pull_request` once I can confirm it works. My final PR will not have this.

Reply:
Sorry, `test` is an overloaded term here, I mean, specifically testing this draft PR to make sure it works before I can submit the PR with only the cron as the trigger.

Reply:
Got it, thanks!